web/gui/src/dashboard.js/units-conversion.js

```diff
@@ -18,13 +18,27 @@ NETDATA.unitsConversion = {
             'Kpps': 1000,
             'Mpps': 1000000
         },
+        'bits/s': {
+            'bits/s': 1,
+            'kilobits/s': 1000,
+            'megabits/s': 1000000,
+            'gigabits/s': 1000000000,
+            'terabits/s': 1000000000000
+        },
         'kilobits/s': {
             'bits/s': 1 / 1000,
             'kilobits/s': 1,
             'megabits/s': 1000,
             'gigabits/s': 1000000,
             'terabits/s': 1000000000
         },
+        'bytes/s': {
+            'bytes/s': 1,
+            'kilobytes/s': 1024,
+            'megabytes/s': 1024 * 1024,
+            'gigabytes/s': 1024 * 1024 * 1024,
+            'terabytes/s': 1024 * 1024 * 1024 * 1024
+        },
```

Comment:
should these be `B/s`, `KiB/s`, `MiB/s`, `GiB/s`, etc?

Reply:
I thought we postponed the Kib/MiB etc units until the collectors are updated.